flowermd/library/systems.py

```diff
@@ -1,3 +1,37 @@
 """Examples for the Systems class."""
 
 # This is a placeholder for any class that inherits from base.system
+import numpy
+from scipy.spatial.distance import pdist
+
+
+class SingleChainSystem(System):
```

Comment:
We'll have to import the `System` class if we are using it in this new python file.